thunder/__init__.py

```diff
@@ -1014,6 +1014,16 @@ def last_compile_options(fn: Callable, /) -> None:
         print(f"\t{option}")
 
 
+def print_auto_registered_torch_ops(fn: Callable, /) -> set[str] | None:
```

Comment:
"print" suggest that the operation would actually print the names. Maybe a more accurate name would be `get_auto_registered_torch_op_names`?